decidim-conferences/app/views/decidim/conferences/conference_program/_program_meeting.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= start_time.to_s(:time) %> - <%= end_time.to_s(:time) %>
   </div>
   <div class="column medium-10 programme-title">
-      <%= link_to present(meeting).title, resource_locator(meeting).path %>
+      <%= link_to decidim_html_escape(present(meeting).title).html_safe, resource_locator(meeting).path %>
```

Comment:
In `develop` we're doing decidim_sanitize: https://github.com/decidim/decidim/blob/3dab60cb3a2ec643fd998e19711aadf067f83eb7/decidim-conferences/app/views/decidim/conferences/conference_program/_program_meeting.html.erb#L2


```suggestion
      <%= link_to decidim_sanitize(present(meeting).title), resource_locator(meeting).path %>
```